_translator-files/po/de/Installation-for-Android.po

```diff
@@ -41,7 +43,7 @@ msgstr "Installation auf Android"
 #. type: Plain text
 #: ../wiki/en/Installation-for-Android.md:9
 msgid "{% include breadcrumb.html root=\"Using Jamulus\" branch1=\"Getting Started\" branch1-url=\"Getting-Started\" %}"
-msgstr "{% include breadcrumb.html root=\"Jamulus Nutzen\" branch1=\"Erste Schritte\" branch1-url=\"Getting-Started\" %}"
+msgstr "{% include breadcrumb.html root=\"Using Jamulus\" branch1=„Setup - Erste Schritte mit Jamulus“ branch1-url=\"Getting-Started\" %}"
```

Comment:
The double quotes in `„Setup - Erste Schritte mit Jamulus“` should be changed to regular " quotes. This is causing an error and preventing the website from displaying the DE version.

Also - "Using Jamulus" should be translated.

Reply:
Replaced in all files „   ->   \"
Seems to be a POEDITOR issue
Noted

Reply:
In other parts of the text it's not a problem if you use those quotes, but these bits expect a specific syntax.

Poedit will only display what you write in it, it won't attempt to replace anything. My guess is keyboard language settings.